### PR TITLE
Add Data Feed Documentation

### PR DESCRIPTION
**Pull request by nicbaz**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -2,11 +2,10 @@
 title: Kaiko API Platform Reference
 language\_tabs:
   - curl
-toc_footers:
+toc\_footers:
```
Comment (Legogris) [inline source/index.html.md]: Why this change? I think this will break the sidebar bottom links.

**Comment by nicbaz** [comment]
> It is fine to use meta data in branch names but please combine it with a descriptive name `DEV2-290` => `DEV2-290-add-data-feed-docs`

If the JIRA integration works then you would have a link to the full ticket, I would argue against this since issues can be renamed and it's cumbersome to create/maintain.

File: source/index.html.md (hunk 1 above)
Comment (nicbaz) [reply source/index.html.md]: Sorry, Markdown parser was complaining. Fixing it for the other header variable.

**Comment by Fonsan** [comment]
@nicbaz working with id:s is a pain when versioning locally. It would not need to be exactly mirrored to the jira issue title but just give an indication. I suppose the integration could be setup to track the first part or does it need deterministic branch names?

**Comment by nicbaz** [comment]
@Fonsan you can put whatever you want as long as it starts with the issue name (per their documentation).
